src/Components/SignatureSideBar/Components/Class101IntroductionBanner.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { Icon, Colors } from '@class101/ui';
+
+import styled from "styled-components";
+
+function Class101IntroductionBanner() {
+  return (
+    <Class101IntroductionBannerContainer>
```

Comment:
```suggestion
    <BannerContainer>
```
1. 어차피 컴포넌트명을 통해 알 수 있기 때문에 과도하게 이름이 길어질 필요 없을거 같습니다
2. 필요 없는 부분이면 그냥 지워주세요